### ISSUE TITLE
TS: No 'Connector' attribute in 'Defaults'

### ISSUE THREAD

**Issue by ledicjp** [jsplumb/jsplumb#744]
Hello everyone,
I just wanted to ask if the "Connector" attribute is missing from the "Defaults" interface intentionally?
![image](https://user-images.githubusercontent.com/12116471/40483389-1ad77a74-5f58-11e8-81b1-96653a9ef13f.png)

##### For example I can't say:

```
this.jsPlumbInstance = jsPlumb.getInstance( // _defaults: {
                PaintStyle: {
                    strokeWidth: 1,
                    stroke: '#212121',
                },
                Endpoint: "Blank",
                Connector: // this doesn't exist in Defaults, therefore it throws errors
                ConnectionOverlays: [
                    ["Arrow", { location: 1 }],
                ],
                ConnectionsDetachable: false,
                Anchor: "Continuous"
            });
```

##### But this works perfectly fine (using the importDefaults method that is not typed afterwards):

```
this.jsPlumbInstance = jsPlumb.getInstance( //_defaults: {
                PaintStyle: {
                    strokeWidth: 1,
                    stroke: '#212121',
                },
                Endpoint: "Blank",
                ConnectionOverlays: [
                    ["Arrow", { location: 1 }],
                ],
                ConnectionsDetachable: false,
                Anchor: "Continuous"
            });
this.jsPlumbInstance.importDefaults({Connector: "Flowchart"});
```

I'm using jsPlumb 2.7.0 along with Angular6, but this behavior goes back to 2.6.2 when you added the typings I think, not sure.

**Comment by sporritt** [comment]
its an oversight. we'll get a 2.7.1 release out with an update shortly.

**Comment by sporritt** [comment]
fixed in 2.7.1, just released.
